AutomatedTesting/Gem/PythonTests/Performance/tests/EditorLevelLoading_base.py

```diff
@@ -0,0 +1,75 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import time
+
+# fmt: off
+class Tests():
+    enter_game_mode = ("Entered game mode", "Failed to enter game mode")
+    exit_game_mode  = ("Exited game mode",  "Couldn't exit game mode")
+# fmt: on
```

Comment:
Class appears to provide nothing, the tuples can simply be defined as module-level constants.